### DISCUSSION TITLE
PublishFormat.pfHTML requires .htm extention, not .html extention

### DISCUSSION THREAD

**Discussion by quiret** [MicrosoftDocs/office-developer-client-docs#672] [Q&A]
![image](https://user-images.githubusercontent.com/7682485/120858146-a5063580-c582-11eb-9fc7-9ef2bbc21d2f.png)

But using .html as file extention causes a 0x8004201A HRESULT at the Publish function, while using .htm instead works just fine. Thus the documentation is misleading.

Look here for more details: https://stackoverflow.com/questions/26363784/comexception-when-converting-onenote-into-html-but-it-works-for-pdf

**Comment by lindalu-MSFT** [comment]
@quiret Which file is this found in?
